pump/server.go

```diff
@@ -597,6 +600,23 @@ func (s *Server) BinlogByTS(w http.ResponseWriter, r *http.Request) {
 		fmt.Fprint(w, "\n\n PrewriteValue: \n")
 		fmt.Fprint(w, prewriteValue.String())
 	}
+
+	if len(binlog.PrewriteKey) > 0 {
+		tikvStorage := s.tiStore.(tikv.Storage)
+		healper := storage.Helper{
+			Store:       tikvStorage,
+			RegionCache: tikvStorage.GetRegionCache(),
+		}
+
+		resp, err := healper.GetMvccByEncodedKey(binlog.PrewriteKey)
```

Comment:
we can extract one function `queryTransactionStatus`, then just call it. the implementation output too many unless and  old mvcc records

Reply:
for debug just keep all info?